src/pages/clients-list/forms/ClientTemplateForm.js

```diff
@@ -605,6 +587,37 @@ const ClientTemplateForm = ({ setErrorMessage, recordId, _labels, plantId, maxAc
       setProfessionFilterStore(filteredList)
     }
   }
+  const { userId } = JSON.parse(window.sessionStorage.getItem('userData'))
+
+  const onClose = async () => {
+    const values = clientIndividualFormik.values
+    try {
+      const { record: cashAccountRecord } = await getRequest({
+        extension: `SY.asmx/getUD`,
+        parameters: `_userId=${userId}&_key=cashAccountId`
+      })
+
+      const data = {
+        recordId: values?.recordId || null,
```

Comment:
what's the difference between clientId and recordId ? 
this is a recordId for the client so IF we need both fields THEN they should have the same value.